classes/query/QueryDAO.php

```diff
@@ -138,7 +138,8 @@ public function countOpenPerStage(int $submissionId, ?array $participantIds = nu
             ->toArray();
 
         return collect(Application::get()->getApplicationStages())
-            ->mapWithKeys(fn ($stageId, $key) => [$stageId => $counts[$stageId] ?? 0]);
+            ->mapWithKeys(fn ($stageId, $key) => [$stageId => $counts[$stageId] ?? 0])
+            ->toArray();
```

Comment:
Is this strictly to satisfy the documented return type? If so, I'd rather not merge changes like this so late in the 3.4.0 release. Once we have a `stable-3_4_0` branch distinct from `main`, we can start adding explicit type declarations to `main` and scaring up any bugs that get introduced early in the dev cycle.

Reply:
I'd rather we just left it as is, since it's already deprecated.